app/main.js

```diff
@@ -11,27 +12,104 @@ global.gMapping = require('./mapping');
 
 global.win;
 let defaultFilePath = path.join(app.getPath('desktop'), app.getName() + ' Files');
+let template = [{
+  label: 'Edit',
+  submenu: [{
+    label: 'Undo',
+    accelerator: 'CmdOrCtrl+Z',
+    role: 'undo'
+  }, {
+    label: 'Redo',
+    accelerator: 'Shift+CmdOrCtrl+Z',
+    role: 'redo'
+  }, {
+    type: 'separator'
+  }, {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    role: 'cut'
+  }, {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    role: 'copy'
+  }, {
+    label: 'Paste',
+    accelerator: 'CmdOrCtrl+V',
+    role: 'paste'
+  }, {
+    label: 'Select All',
+    accelerator: 'CmdOrCtrl+A',
+    role: 'selectall'
+  }]
+}, {
+  label: 'Window',
+  role: 'window',
+  submenu: [{
+    label: 'Minimize',
+    accelerator: 'CmdOrCtrl+M',
+    role: 'minimize'
+  }]
+}, {
+  label: 'Help',
+  role: 'help',
+  submenu: [{
+    label: 'sw-exporter on GitHub',
+    click: function () {
+      electron.shell.openExternal('https://github.com/Xzandro/sw-exporter')
+    }
+  }]
+}]
+
+
+if ("darwin" === process.platform) {
```

Comment:
REALLY small comment. Can you do `'process.platform === darwin'`? Looks better to me.